catalog.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Дизайн-студия "Нердс"</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
+  <link href="normalize.css" rel="stylesheet">
+  <link href="css/style.css" rel="stylesheet">
+</head>
+
+<body class="page-body catalog-page">
+  <header class="main-header">
+    <nav class="main-navigation">
+      <a class="main-header-logo">
+        <img href="index.html" src="img/nerds-logo.svg" width="160" height="49" alt="Логотип дизайн-студии «Нердс»">
+      </a>
+      <div class="main-navigation-wrapper">
+        <div class="container">
+          <ul class="site-navigation">
+            <li><a href="blank.html">Студия</a></li>
+            <li><a href="blank.html">Клиенты</a></li>
+            <li><a href="blank.html">Магазин</a></li>
```

Comment:
```suggestion
            <li><a>Магазин</a></li>
```
так как это текущая страница.